### DISCUSSION TITLE
Hue motion sensor - stopped working but is connected - why?

### DISCUSSION THREAD

**Discussion by ghost** [Koenkk/zigbee2mqtt#15060] [Devices]
Can anyone help me understand what the flashing '%' means or what is happening here?

The "Orangery motion" sensor was working perfectly until today.  I have changed the batteries, reset it, removed and repaired it and I get the same flashing '%' every time (also nothing is reported - motion, temp, lux).

![Screenshot 2022-11-17 at 21 40 20](https://user-images.githubusercontent.com/34644672/202565619-262f30e8-7db0-41fa-9bc1-5ecbfbd20e49.png)

thanks.

**Comment by jdhgmann04** [comment]
Interesting thing happened to me with the Aqara devices which i think i have resolved with a workaround? maybe? After using MQTT to listen for zigbee2mqtt/"sensorname"# i was able to determine the sensors were communicating, just not updating on the frontend OR Z2M. Also i noticed any Aqara devices running through a router were the culprit, zero issues with direct connected devices to the USB coordinator. After repairing repeated times and trying with router connection and without it seems to be that simply renaming the device name to something else in Z2M and renaming back corrects the issue with the frontend and the devices start operating correctly. Maybe worth a shot for you? The routers that i'm having issues with are the Sonoff S31ZB outlets.

**Reply by jdhgmann04** [reply]
Also the flashing % means it is not getting the battery status or it is dead. In my experience it usually means the sensor is not communicating properly and the above mentioned corrected the issue.

**Comment by ghost** [comment]
thanks, @jdhgmann04 and sorry for the late reply.
